tests/session_auth/test_session.py

```diff
@@ -370,3 +372,16 @@ def test_wrong_cookie_value(self):
             response.json(),
             {"is_unauthenticated_user": True, "is_authenticated": False},
         )
+
+
+class TestCleanSessions(TestCase):
+    def setUp(self):
+        SessionsBase.create_table().run_sync()
```

Comment:
It would be nice here if we could add a session to the database table.

Reply:
@dantownsend I agree with you. I'll do it tomorrow.

Reply:
Thanks!